azure/table_azure_compute_virtual_machine.go

```diff
@@ -47,6 +47,12 @@ func tableAzureComputeVirtualMachine(_ context.Context) *plugin.Table {
 				Description: "The friendly name that identifies the virtual machine.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "virtual_machine_scale_set",
```

Comment:
This is necessary to distinguish standard VMs from VMs managed by VMSS in flexible orchestration mode.